03_launch_mgmt_cluster.sh

```diff
@@ -14,6 +14,18 @@ export IRONIC_HOST="${CLUSTER_URL_HOST}"
 export IRONIC_HOST_IP="${CLUSTER_PROVISIONING_IP}"
 export REPO_IMAGE_PREFIX="quay.io"
 
+# -k is for keepalived
+BMO_IRONIC_ARGS=(-k)
+if [[ "${IRONIC_TLS_SETUP}" == "true" ]]; then
```

Comment:
Nit: [[ is not necessary here, and convention in this file is single [.

(== is also not necessary, but is a convention in the file, so ignoring it)

Reply:
Thanks, fixed